service/awsconfig/v5/resource/loadbalancer/desired.go

```diff
@@ -4,7 +4,12 @@ import (
 	"context"
 )
 
-// TODO
+// GetDesiredState returns an empty state as this resource only implements
+// deletion.
 func (r *Resource) GetDesiredState(ctx context.Context, obj interface{}) (interface{}, error) {
-	return nil, nil
+	emptyState := &LoadBalancerState{
+		LoadBalancerNames: []string{},
+	}
+
+	return emptyState, nil
```

Comment:
IMO desired state should just be nil in case it is empty. Based on my experience with resource implementations it is way easier to work with this than having some types with some values because you then need to identify what an empty state means, which is just more complicated than comparing against `nil`.

Reply:
Fixed.